backend/Dockerfile

```diff
@@ -86,12 +92,12 @@ FROM base as production
 # Copy "binary"-files from build image
 COPY --from=build ${DOCKER_WORKDIR}/dist ./dist
 COPY --from=build ${DOCKER_WORKDIR}/node_modules ./node_modules
-# Copy static files
+# Copy static files # Wolle comment overfluid here?
 # TODO - externalize the uploads so we can copy the whole folder
 COPY --from=build ${DOCKER_WORKDIR}/public/img/ ./public/img/
 COPY --from=build ${DOCKER_WORKDIR}/public/providers.json ./public/providers.json
 # Copy package.json for script definitions (lock file should not be needed)
 COPY --from=build ${DOCKER_WORKDIR}/package.json ./package.json
```

Comment:
yarn install --productoin

Reply:
`WORKDIR` is already set, so you can do:
```
COPY --from=code ./public/img ./public/providers.json ./package.json ./
```

Reply:
To use `yarn install --productoin` here is unfortunately not working at the moment. @Mogge @roschaefer 
There are necessary packages missing. @ulfgebhardt created the  issue #4471 for that.

Thanks for the cool hint @roschaefer . I changed this.